internal/controller/olsconfig_controller.go

```diff
@@ -47,17 +49,19 @@ const (
 // OLSConfigReconciler reconciles a OLSConfig object
 type OLSConfigReconciler struct {
 	client.Client
-	Scheme     *runtime.Scheme
-	logger     logr.Logger
-	stateCache map[string]string
-	Options    OLSConfigReconcilerOptions
+	Scheme            *runtime.Scheme
+	logger            logr.Logger
+	stateCache        map[string]string
+	Options           OLSConfigReconcilerOptions
+	NextReconcileTime time.Time
 }
 
 type OLSConfigReconcilerOptions struct {
 	LightspeedServiceImage      string
 	LightspeedServiceRedisImage string
 	ConsoleUIImage              string
 	Namespace                   string
+	ReconcileInterval           int
```

Comment:
I don't have any particular questions for this PR.
But maybe it's worth using `time.Duration` instead of `int` so as not to do additional type conversion
Of course except in a case when we really plan to use it as an `int`

Reply:
updated to use `time.Duration` now :) thanks!